ext/setup/main.php

```diff
@@ -407,11 +406,8 @@ public function onConfigSave(ConfigSaveEvent $event): void
                 }
             }
         }
+        $config->save();
         log_warning("setup", "Configuration updated");
-        foreach (\Safe\glob("data/cache/*.css") as $css_cache) {
-            unlink($css_cache);
-        }
-        log_warning("setup", "Cache cleared");
```

Comment:
These files haven't lived here for a while, and cache-clearing shouldn't be needed since the theme name is now included as part of the hash filename